docs/debug.md

```diff
@@ -415,107 +415,111 @@ $ echo 0 > options/graph-time
-ERROR:   [0x0] TEE-CORE:  arch: arm  load address: 0x103000  ctx-idr: 1
-ERROR:   [0x0] TEE-CORE:  stack: 0x100000 10240
-ERROR:   [0x0] TEE-CORE:  region 0: va 0x100000 pa 0xe31e000 size 0x3000 flags rw-
-ERROR:   [0x0] TEE-CORE:  region 1: va 0x103000 pa 0xe200000 size 0x2e000 flags r-x
-ERROR:   [0x0] TEE-CORE:  region 2: va 0x131000 pa 0xe22e000 size 0xa000 flags r--
-ERROR:   [0x0] TEE-CORE:  region 3: va 0x13b000 pa 0xe238000 size 0xe6000 flags rw-
-ERROR:   [0x0] TEE-CORE:  region 4: va 0 pa 0 size 0 flags ---
-ERROR:   [0x0] TEE-CORE:  region 5: va 0 pa 0 size 0 flags ---
-ERROR:   [0x0] TEE-CORE:  region 6: va 0 pa 0 size 0 flags ---
-ERROR:   [0x0] TEE-CORE:  region 7: va 0 pa 0 size 0 flags ---
-ERROR:   [0x0] TEE-CORE: Call stack:
-ERROR:   [0x0] TEE-CORE:  0x00105bdc
-ERROR:   [0x0] TEE-CORE:  0x00121bc9
-ERROR:   [0x0] TEE-CORE:  0x00121d1b
-ERROR:   [0x0] TEE-CORE:  0x00121e6f
-```
-
-The above dump was triggered by the TA trying to write to a global const
-variable. Such variables are stored in the `.rodata` section of the ELF binary,
-and loaded into a read-only section of memory.
-
-OP-TEE provides a helper script called `symbolize.py` to facilitate the
-analysis of such issues. It is located in the OP-TEE OS source tree, and is
-also copied to the TA development kit. Whenever you are confronted with an error
-message reporting a serious error and showing a "Call stack:" line, you may use
-the symbolize script.
+  read or write permission fault).
+
+Here is an example of a panic in a 32-bit Trusted Application, running on a
+32-bit TEE core (QEMU):
+
+```
+E/TC:0 TA panicked with code 0x0
+E/TC:0 Status of TA 484d4143-2d53-4841-3120-4a6f636b6542 (0xe07ba50) (active)
+E/TC:0  arch: arm  load address: 0x101000  ctx-idr: 1
+E/TC:0  stack: 0x100000 4096
+E/TC:0  region 0: va 0x100000 pa 0xe31d000 size 0x1000 flags rw-
+E/TC:0  region 1: va 0x101000 pa 0xe300000 size 0xf000 flags r-x
+E/TC:0  region 2: va 0x110000 pa 0xe30f000 size 0x3000 flags r--
+E/TC:0  region 3: va 0x113000 pa 0xe312000 size 0xb000 flags rw-
+E/TC:0  region 4: va 0 pa 0 size 0 flags ---
+E/TC:0  region 5: va 0 pa 0 size 0 flags ---
+E/TC:0  region 6: va 0 pa 0 size 0 flags ---
+E/TC:0  region 7: va 0 pa 0 size 0 flags ---
+E/TC:0 Call stack:
+E/TC:0  0x001044a8
+E/TC:0  0x0010ba59
+E/TC:0  0x00101093
+E/TC:0  0x001013ed
+E/TC:0  0x00101545
+E/TC:0  0x0010441b
+E/TC:0  0x00104477
+D/TC:0 user_ta_enter:452 tee_user_ta_enter: TA panicked with code 0x0
+D/TC:0 tee_ta_invoke_command:649 Error: ffff3024 of 3
+D/TC:0 tee_ta_close_session:402 tee_ta_close_session(0xe07be98)
+D/TC:0 tee_ta_close_session:421 Destroy session
+D/TC:0 tee_ta_close_session:447 Destroy TA ctx
+```
+
+The above dump was triggered by the TA when entering an irrecoverable error
+ending up in a `TEE_Panic(0)` call.
+
+OP-TEE provides a helper script called `symbolize.py` to facilitate the analysis
+of such issues. It is located in the OP-TEE OS source tree, and is also copied
```

Comment:
remove coma.

s/copied to/installed with/ ?

Reply:
It's Jerome's wording and I think copied to is fine here, it's not that we are running something like "make install" (so I'll keep the current wording).